### ISSUE TITLE
Add particle confinement proxies

### ISSUE THREAD

**Issue by f0uriest** [PlasmaControl/DESC#522]
such as Nemov's original $\Gamma_C$ [link](https://itp.tugraz.at/AG/APP/PLASMA/PUBLIC/OWN_PUBLICATIONS/PAPERS/Nemov2008-052501.pdf) or the newer $\Gamma_\alpha$ [link](https://iopscience.iop.org/article/10.1088/1741-4326/ac2994/pdf?casa_token=zeIpampigfkAAAAA:HrpLe7n6fwJAICFhvT5N9v8eV0lFEcyXXNnp8CzaiULodzT-AMlOqTnDFYPIMq25i4pXpgbUBxwKFZ4)

**Comment by rahulgaur104** [comment]
Kaya and I will try to finish the bounce averaging PR this week. Once that is done, we can all the different types of gammas and epsilons.

**Comment by dpanici** [comment]
`jc/gammaC` is the branch where the code from our intern Jackson is from his implementation, not completely finished but all of the main equations are coded up there so will be a good start, we would just need to take those and put them into Kaya's framework for the field-line bounce integrals

**Comment by rahulgaur104** [comment]
I think it would be much easier for me to write my own GammaC code than to understand and rewrite Jackson's code. Once we have a simpler version ready, we can borrow specific parts like vectorization etc. from his branch.

**Comment by unalmis** [comment]
Maybe just make new issue for new proxies like we did with max j etc since gamma_c is done.
